### PR TITLE
Reimplemented Settings in Swift 

### LINKED ISSUE #35: Reimplement Settings in Swift

### PR DESCRIPTION
**Pull request by lennet**
This PR contains a reimplementation of the Settings-screen with the new Design. 

fixes #35

### CODE REVIEW

**Comment by zutrinken** [comment]
I'm in love! One thing though is the line-height and spacing. Fixing this would be a blast.

**Comment by lennet** [comment]
> I'm in love! One thing though is the line-height and spacing. Fixing this would be a blast.
> 
> 

I updated the Github cell

bildschirmfoto 2019-02-01 um 13 46 36

Should I also increase the height for the other cells ?

**Comment by zutrinken** [comment]
> Should I also increase the height for the other cells ?

Sure. In the concept all cells have a minimum height of 56px.

**Comment by lennet** [comment]
bildschirmfoto 2019-02-01 um 14 58 49

File: CriticalMass/SettingsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        }
+
+        var models: [Model] {
+            switch self {
+            case .preferences:
+                return [Model(title: NSLocalizedString("GPS", comment: ""), action: .none)]
+            case .github:
+                return [Model(title: nil, action: .open(url: URL(string: "https://github.com/criticalmaps/criticalmaps-ios")!))]
+            case .info:
+                return [Model(title: NSLocalizedString("settings.website", comment: ""), action: .open(url: URL(string: "http://www.criticalmass-berlin.org")!)),
```
Comment (zutrinken) [inline CriticalMass/SettingsViewController.swift]: https://www.criticalmaps.net

File: CriticalMass/SettingsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                        Model(title: NSLocalizedString("settings.twitter", comment: ""), action: .open(url: URL(string: "https://twitter.com/criticalmaps/")!)),
+                        Model(title: NSLocalizedString("settings.facebook", comment: ""), action: .open(url: URL(string: "https://www.facebook.com/pages/Critical-Mass-Berlin/74806304846")!))]
```
Comment (zutrinken) [inline CriticalMass/SettingsViewController.swift]: https://www.facebook.com/criticalmaps

File: CriticalMass/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -105,14 +105,27 @@
...
 
-"settings.openSource" = "This is an open source project. Feel free to support us with coding for iOS or Web:";
+/* No comment provided by engineer. */
+"settings.opensource.title" = "Vermisst du Funktionen oder willst einen Bug fixen?";
+
+/* No comment provided by engineer. */
+"settings.opensource.detail" = "Critical Maps ist ein Open Source Projekt und wir feuen uns über Entwickler*innen die an Critical Maps mitarbeiten wollen.";
+
+/* No comment provided by engineer. */
+"settings.opensource.action" = "PROJEKT ANSEHEN";
```
Comment (headione) [inline CriticalMass/de.lproj/Localizable.strings]: Can we change to normal case "Projekt ansehen" and uppercase the string in the code? If design changes, we don't need to touch the strings imo

Comment (lennet) [reply CriticalMass/de.lproj/Localizable.strings]: Yes. Good Idea!

**Pull request merged**